src/main/java/appeng/client/texture/ExtraItemTextures.java

```diff
@@ -36,7 +36,11 @@
 
 	ToolColorApplicatorTip_Dark( "ToolColorApplicatorTip_Dark" ),
 
-	ToolColorApplicatorTip_Light( "ToolColorApplicatorTip_Light" );
+	ToolColorApplicatorTip_Light( "ToolColorApplicatorTip_Light" ),
+
+	WirelessTerminal_Inactive( "ToolWirelessTerminalInactive" ),
+
+	WirelessTerminal_Border( "ToolWirelessTerminalBorder" );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00115)